src/mod_stream_management.erl

```diff
@@ -39,7 +39,11 @@
          remove_stale_smid_h/2
         ]).
 
--type smid() :: base64:ascii_binary().
+
+%% base64:ascii_binary() type is not exported
+-type ascii_binary() :: binary().
```

Comment:
the same question, should we put this type in some generic module (e.g. mongooseim)?